### PR TITLE
Moonquant Capital Tokens

### PR DESCRIPTION
**Pull request by NelsonGaldeman**
Signature:
0xed8f00fcbca751e6f550edfb1c4fe39b7d25183e99c83c202ce65f10b2d999187c3902dfe646359b9fb313773a98420dafc8d969966f58f7d90d96d610a53c7a1b

### CODE REVIEW

File: set.tokenlist.json [hunk 1]
```diff
@@ -8,6 +8,22 @@
   ],
   "timestamp": "2020-10-19T12:32:17.000+00:00",
   "tokens": [
+    {
```
Comment (asoong) [inline set.tokenlist.json:11]: Move these to the bottom of the file.

**Review by asoong — COMMENTED** [review]
Can you update the token logo file names to be downcased and more descriptive?

File: set.tokenlist.json (hunk 1 above)
Comment (NelsonGaldeman) [reply set.tokenlist.json]: Moved!

**Comment by NelsonGaldeman** [comment]
> Can you update the token logo file names to be downcased and more descriptive?

Done!

**Pull request merged**